tiatoolbox/utils/misc.py

```diff
@@ -355,6 +358,53 @@ def contrast_enhancer(img, low_p=2, high_p=98):
     return np.uint8(img_out)
 
 
+def numpy_array_to_table(input_table):
+    """Checks numpy array to be 2 or 3 columns.
+    If it has two columns then class should be assign None.
+
+    Args:
+        input_table (np.ndarray): input table.
+
+    Returns:
+       table (:class:`pd.DataFrame`): Pandas DataFrame with desired features.
+
+    Raises:
+        ValueError: If the number of columns is not equal to 2 or 3.
+
+    """
+    if input_table.shape[1] == 2:
+        out_table = pd.DataFrame(input_table, columns=["x", "y"])
+        out_table["class"] = None
+        return out_table
+
+    if input_table.shape[1] == 3:
+        return pd.DataFrame(input_table, columns=["x", "y", "class"])
+
+    raise ValueError("numpy table should be of format `x, y` or " "`x, y, class`")
+
+
+def assign_unknown_class(input_table):
```

Comment:
Should these be private function?

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/250/commits/4c2e04b86bfccdb876dd639de58b96f72885a20b